### PR TITLE
GitHub Community Checklist

### PR DESCRIPTION
**Pull request by Rohan-cod**
This Pull Request checks the following in the GitHub Community Checklist:

1. Code of Conduct
2. Issue template
3. Pull Request template

The remaining checks, 'Description' and 'Repository admins accept content reports' can only be done by the admins.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
+# Contributor Covenant Code of Conduct
```
Comment (srgothi92) [inline CODE_OF_CONDUCT.md:1]: @rhdolin Would you like to give this a read ? It's stand based on https://www.contributor-covenant.org/ but we should give it a read and see if we required exactly this Code of Conduct.

**Review by srgothi92 — COMMENTED** [review]
👍

**Comment by Rohan-cod** [comment]
@srgothi92, I have removed 'Checklist' and 'Type of change' from the Pull Request template.

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (rhdolin) [reply CODE_OF_CONDUCT.md:1]: @srgothi92 I'm okay starting with this, and we can adapt if necessary as we go along.

**Review by srgothi92 — APPROVED** [review]

**Pull request merged**